config/config_hard_filter_cnv_amp.yaml

```diff
@@ -0,0 +1,13 @@
+filters:
+  # artifacts:
+  #   description: Hard filter variants found in more than 15% of the normal samples
+  #   expression: "(INFO:Twist_AF > 0.15)"
+  #   soft_filter: "False"
+  copy_number:
+    description: Hard filter CNVs with fewer than six copies
+    expression: "(INFO:CORR_CN < 6)"
```

Comment:
At least at Uppsala they want to look at lower freq CNVs but guess we can fix that later

Reply:
Alright! Maybe we can bring this up as a discussion point during today's meeting?